### ISSUE TITLE
Refresh of 0 hides all Fate tabs

### ISSUE THREAD

**Issue by fionawhim** [Sk1mble/fate-core-official#7]
I got surprising behavior starting with the “Blank” actor template.

If the Refresh value is 0, the “Character Sheet” and “Description, Biography, and Extras” tabs don’t appear. Changing it to 1 makes them appear.

<img width="960" alt="Screen Shot 2021-06-19 at 10 22 47 AM" src="https://user-images.githubusercontent.com/260837/122645575-ff59d700-d0e8-11eb-9004-fa7580a4b17c.png">

<img width="956" alt="Screen Shot 2021-06-19 at 10 22 39 AM" src="https://user-images.githubusercontent.com/260837/122645570-fcf77d00-d0e8-11eb-91c1-6932779c6f45.png">

Foundry VTT 0.8.6
Fate Core Official 4.1.3
No installed modules

This was confusing because the Blank template starts at 0. I was also expecting to set mooks and such to have 0 for a refresh.

**Comment by Talaren** [comment]
I checked the code and found the error in the character sheet, there the points are hidden when the value "Refresh" is no longer greater than 0.
`{{#if (gt data.details.fatePoints.refresh 0)}}`
I have the problem that most NPCs don't have Fate points available, mostly only bosses do.
Most NPCs have one to three stress points and a few aspects.

**Comment by Sk1mble** [comment]
Hiya,

This is not a bug but an intended feature... I appreciate that this may not
be the ideal way of doing this as NPCs technically have refresh 0 but it
was a simple way of enabling a very minimal interface for certain
characters in Blood on the Trail.

Refresh only does anything in the system for PC's, as the GM has a pool of
Fate points for the scene rather than the NPC's having their own refresh.

I'll consider if there's an easier way to provide the minimal character
sheet in future, but it would probably involve adding a new data element to
characters and that's something I wanted to avoid.

Richard

**Comment by fionawhim** [comment]
Thanks for the response!

I totally get that refresh on NPCs is pretty meaningless, so it doesn’t matter if they’re left above 0.

My one note would be that things felt really broken when I tried to start with a blank sheet (I was trying to put together a default sheet for a FAE mook). I couldn’t find the headings to add Skills and Aspects so I tried to merge them in from an existing sheet and they still wouldn’t show up. It was by accident that I discovered that changing the Refresh made a difference.

Would it make sense to start Blank with a refresh of 1? (With maybe a “Nothing” default or something w/ refresh of 0?)

Or this could just be a documentation thing.
